BinarySearch.py

```diff
@@ -0,0 +1,47 @@
+"""You're going to write a binary search function.
+You should use an iterative approach - meaning
+using loops.
+Your function should take two inputs:
+a Python list to search through, and the value
+you're searching for.
+Assume the list only has distinct elements,
+meaning there are no repeated values, and
+elements are in a strictly increasing order.
+Return the index of value, or -1 if the value
+doesn't exist in the list."""
+
+
+def binary_search(input_array, value):
+    """Your code goes here."""
+    l = len(input_array)
+    start = 1

```

Comment:
usually we have start as `0`, and end as `len(input_array)`, so the array is always [ start, end),  start included, end not included.